### PR TITLE
 Complete - callbacks, arrays, objects, closure - Zack Hitchcock

### PR DESCRIPTION
**Pull request by zackhitch**

### CODE REVIEW

File: src/callbacks.js [hunk 1]
```diff
@@ -1,34 +1,43 @@
...
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+  const arr = [];
+  for (let i = 0; i < array.length; i++) {
+    if (!arr.includes(array[i])) {
+      arr.push(array[i]);
+    }
+  }
+  cb(arr);
 };
```
Comment (randallwhitlock) [inline src/callbacks.js:42]: Very cool.  Another way you can do this (of course there are a million) in one line of code is to use this:
```
const removeDuplicates = (array, cb) => cb(Array.from(new Set(array)));
```` 
What this does is it invokes Array.from, which is a method on the Array constructor that creates a new array with the values it is called with. 
new Set() is creating a new Set object with the array values that we pass into it. Since Sets only hold unique values, that is what the end result of the array is made with.
So basically we make a new set from our array and then make a new array from our new set. Haha

**Review by randallwhitlock — COMMENTED** [review]
NIce work so far!
